h/static/scripts/annotator/plugin/enhancedanchoring.coffee

```diff
@@ -83,6 +89,37 @@ class Anchor
       # Kill the list if it's empty
       delete @anchoring.anchors[index] unless anchors.length
 
+  # Scroll to this anchor
+  scrollTo: ->
```

Comment:
Why is this called scrollTo, but it takes no arguments? Perhaps take a tip from our jQuery plugin and call it `scrollIntoView`?